common-ui/tslint.json

```diff
@@ -146,8 +143,6 @@
       "Layout"
     ],
     "directive-class-suffix": true,
-    "no-access-missing-member": true,
```

Comment:
I believe these were stale and don't work on the latest Angular - can you confirm that is why these 3 were removed?

Reply:
They're not even in the current tslint.